package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can you delete this file? It's not needed at the root directory.

Reply:
The `package-lock.json` file should be deleted? @fozziethebeat